interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -824,6 +896,19 @@ function reverse_cylinder() {
 <body>
 <?php echo report_header($pid, "web");  ?>
 <br/><br/>
+<?php
+    $row['REFDATE'] = oeFormatShortDate($data['date']);
```

Comment:
You should not be converted to the localized date here. I think that will break the strtotime calls below. Converting it to localized date after the conversion as you do below makes more sense.